docs/conductor/getting-started.md

```diff
@@ -0,0 +1,88 @@
+- get access to Conductor from [Private Packagist Support](mailto:contact@packagist.com)
+- add a workflow to your Continuous Integration Platform to run Composer updates
+- verify the setup
+
+
+## First steps
+
+Once you have a Private Packagist Subscription and Conductor is enabled for your subscription, log into your Private Packagist organization and click on the Updates tab in the main navigation.  
+Conductor will list all Packages with a composer.lock file and the Private Packagist repository added to the composer.json.    
+If you do not see your package, follow the instructions on your organization page to add the custom repository to the composer.json of your package.  
+
+Most convenient way to set up Conductor is to configure your CI in the Private Packagist UI. The manual steps are outlined below:
+
+## Create a workflow on your CI
+
+### GitHub Actions
+
+Create a new GitHub Actions workflow in `.github/workflows/dependency-update.yaml` of your package repository using the template below:
+
+CONDUCTOR_GITHUB_ACTIONS_WORKFLOW
+
+- adjust the PHP Version used in the "Install PHP" step
+- commit and push the workflow to your main branch of your package repository
+- review the commit and push method: the workflow needs to be able to push commits to a new branch
+
+Create a secret `COMPOSER_AUTH` with the Composer authentication configuration [as described here](https://getcomposer.org/doc/articles/authentication-for-private-packages.md#authentication-using-the-composer-auth-environment-variable) to access Private Packagist.  
+We recommend to create a dedicated authentication token with update access. You can copy n paste the contents for the secret from the Private Packagist UI while creating the token in "Settings" -> "Authentication Tokens". Remove the single quotes around the value.
+
+![Create Authentication Token](/Resources/public/img/docs/conductor/authentication-token.png)
+
+The contents of the variable should look like 
+
+```json
+{"http-basic": {"repo.packagist.com": {"username": "token", "password": "packagist_out_73a81c7eb525b13b6bc22a410b2146e78a38b324f609bf1158c583c704f64cbdd349" }}}
+```
+
+Conductor needs to verify your setup before you can [start receiving Pull Requests](#how-scheduling-works). 
+
+## How scheduling works
+
+- Go to your package on the updates tab in your Private Packagist organization 
+- Click on the name of your package
+
+The list shows groups of all available updates to be scheduled. Each group of updates is called a task. Conductor will schedule only one task at a time. All others are waiting for the task on top of the list to be successful or paused.  
+Once Conductor schedules a task it sends a payload to your CI Platform that triggers the workflow you just added. The payload contains the commands Composer will run to update a group of dependencies from your package.
+
+The workflow consists of several steps:
+
+- code checkout
+- php setup
```

Comment:
```suggestion
- Setup PHP
```

If you create lists, try to make sure all entries follow the same structure. Mixing things like `code checkout` with `Send the status of the workflow to Private Packagist via cURL` makes it a bit hard to read. Think in cases like this it's nice to have full sentences that start with a verb.

Reply:
Thanks, that's a good hint. Will add this to my writing checklist